DOCS/man/mpv.rst

```diff
@@ -371,6 +371,79 @@ To use this feature, you need to fill the ``menu-data`` property with menu
 definition data, and add a keybinding to run the ``context-menu`` command,
 which can be done with a user script.
 
+Image Bindings
+--------------
+
+Several key bindings are changed when viewing images:
+
+LEFT, RIGHT, UP, DOWN, h, j, k, l
+    Pan the image when it is larger than the window.
+
+Ctrl+LEFT, Ctrl+RIGHT, Ctrl+UP, Ctrl+DOWN, Ctrl+h, Ctrl+j, Ctrl+k, Ctrl+l
+    Align the image to one of the boundaries of the window when it is larger
+    than the window.
+
+= and -
+    Change the zoom.
+
++ and _
+    Change the zoom slowly.
+
+0
+    Reset the zoom.
+
+u
+    Toggle between showing the image unscaled in its original dimensions and
+    fitting it to the window.
+
+o
+    Fill the window with the image.
+
+s
+    Toggle between showing images for 5 seconds in a slideshow and keeping them
+    open forever.
+
+n
+    Go to the next playlist entry.
+
+p
+    Go to the previous playlist entry.
+
+N
+    Go to the next sub-playlist (e.g. directory or achive).
+
+P
+    Go to the previous sub-playlist.
+
+g-g
+    Go to the first playlist entry.
+
+G
+    Go to the last playlist entry.
+
+]
+    Skip 10 playlist entries forwards.
+
+[
+    Skip 10 playlist entries backwards.
```

Comment:
oddly specific. How does 10 number were decided?

Reply:
Anyone finds these useful? Else I will remove them later, as suggested by llyyr.

Reply:
Repurposed brackets to control `--image-display-duration` because brackets control video speed and `--image-display-duration` is the equivalent of speed for images. This was requested in https://github.com/mpv-player/mpv/issues/12496. In that same issue it was suggested to bind page up to `add playlist-pos 10`, that is the best option if someone still wants that binding.